core/controllers/admin.py

```diff
@@ -671,21 +670,34 @@ def _load_dummy_new_structures_data(self) -> None:
             topic_1 = topic_domain.Topic.create_default_topic(
                 topic_id_1, 'Dummy Topic 1', 'dummy-topic-one', 'description',
                 'fragm')
-            topic_1.skill_ids_for_diagnostic_test = [skill_id_1]
-            topic_1.thumbnail_filename = 'thumbnail.svg'
-            topic_1.thumbnail_bg_color = '#C6DCDA'
-            topic_1.subtopics = [
-                topic_domain.Subtopic(
-                    1, 'Title', [skill_id_1], 'image.svg',
-                    constants.ALLOWED_THUMBNAIL_BG_COLORS['subtopic'][0], 21131,
-                    'dummy-subtopic-three')]
-            topic_1.next_subtopic_id = 2
-            topic_1.add_canonical_story(story_id)
-
-            topic_2 = topic_domain.Topic.create_default_topic(
-                topic_id_2, 'Empty Topic', 'empty-topic', 'description',
-                'fragm')
 
+            topic_1.update_meta_tag_content('dummy-meta')
+            svg_file = open(
```

Comment:
I think we normally do this within a with block: "with open(...) as ...". See other code that uses open() and please follow the existing patterns.

Reply:
done